src/circuit/gadget/utilities/lookup_range_check.rs

```diff
@@ -95,7 +108,7 @@ impl<F: FieldExt + PrimeFieldBits, const K: usize> LookupRangeCheckConfig<F, K>
 
             // shifted_word = word * 2^{K-s}
             //              = word * 2^K * inv_two_pow_s
-            vec![q_lookup_bitshift * (word * two_pow_k * inv_two_pow_s - shifted_word)]
+            vec![q_bitshift * (word * two_pow_k * inv_two_pow_s - shifted_word)]
```

Comment:
Potential layout optimization: https://github.com/zcash/orchard/pull/162/files#r677515562 . That optimization only makes sense if we're trying to optimize all queries to only use *cur* and *next*, so it can be done later.